### DISCUSSION TITLE
How to fix dead link for PR?

### DISCUSSION THREAD

**Discussion by niceplugin** [vuejs/router#1717] [Help and Questions] [answered]
Created two PRs.
[#1715](https://github.com/vuejs/router/pull/1715)
[#1716](https://github.com/vuejs/router/pull/1716)

I haven't touched anything related to the link.

What should I do? I've never had this problem before and it's very disconcerting.

**Comment by posva** [comment, marked as answer]
it's not your code. Rebase against the main branch
